tests/test_ldap.py

```diff
@@ -0,0 +1,47 @@
+from .base import BaseTestCase
+
+
+class TestUserLDAP(BaseTestCase):
+
+    SUCCESS_CODE = 200
+
+    def setUp(self):
+        super(TestUserLDAP, self).setUp()
+        self.nxc.enable_app('user_ldap')
+
+    def test_crud_ldap_config(self):
+        res = self.nxc.create_ldap_config()
+        assert res['ocs']['meta']['statuscode'] == self.SUCCESS_CODE
+        config_id = res['ocs']['data']['configID']
+
+        # test get config by id
+        res = self.nxc.get_ldap_config(config_id)
+        assert res['ocs']['meta']['statuscode'] == self.SUCCESS_CODE
+        config_data = res['ocs']['data']
+
+        # test edit config
+        param_to_change = "ldapPagingSize"
+        old_param_value = config_data[param_to_change]
+        new_param_value = 777
+        assert old_param_value != new_param_value
+        res = self.nxc.edit_ldap_config(config_id, data={param_to_change: new_param_value})
+        assert res['ocs']['meta']['statuscode'] == self.SUCCESS_CODE
+        new_config_data = self.nxc.get_ldap_config(config_id)['ocs']['data']
+        assert str(new_config_data[param_to_change]) == str(new_param_value)
```

Comment:
That's interesting, what is the point of the conversion? You treat `new_config_data[param_to_change]` as integer earlier in the test.

Reply:
You right, didn't notice that at first it was integer.
After request it set `ldapPagingSize` as string, even though I sent integer there. Looks like it happens because of `x-www-form-urlencoded` Content-type which is required.
```
(Pdb) config_data[param_to_change]
500
(Pdb) new_param_value
777
(Pdb) new_config_data[param_to_change]
'777'
```